### PR TITLE
Fix #518: show server errors in all action arguments

### LINKED ISSUE #518: Required fileds are not highlighted when trying to save an incomplete recipe
**[Affected versions]:**
- Firefox Nightly 83.0a1, Build ID 20201014214248

**[Affected Platforms]:**
- Windows 10

**[Prerequisites]:**
- Have VPN access to the staging environment.
- Have the NDT addon (version 2.1.1) installed on a new profile.

**[Steps to reproduce]:**
1. Open the Firefox profile from prerequisites and click the NDT button from the Browser Toolbar.
2. Switch to the `Stage` environment and click the `Create Recipe` button.
3. Select any action from the `Action` dropdown.
4. Click the `Save` button and leave a comment.
5. Click the `Save` button again and observe the behavior.

**[Expected results]:**
- All required fields are marked as such.

**[Actual results]:**
- `An Error Occurred: {"name":["This field is required."],"status":400}` is displayed but **NO** fields are highlighted.

**[Notes]:**
- Attached a screen recording of the issue.
required fields are not highlighted

### PR DESCRIPTION
**Pull request by leplatrem**
Follow-up of #616

### CODE REVIEW

File: content/components/recipes/form/arguments/PreferenceRollback.tsx [hunk 1]
```diff
@@ -37,12 +38,15 @@ export default function PreferenceRollback(): ReactElement {
...
-    />
+    <>
+      <SelectField
+        searchable
+        data={rolloutData}
+        label="Rollout"
+        name="rolloutSlug"
+        renderMenuItem={rolloutMenu}
+      />
+      <ServerErrors field="arguments.slug" />
```
Comment (leplatrem) [inline content/components/recipes/form/arguments/PreferenceRollback.tsx]: This is redundant with what is done in `SelectField` but the field name is `slug` and not `rolloutSlug`

Comment (mythmon) [reply content/components/recipes/form/arguments/PreferenceRollback.tsx]: Hm. Why is it `slug` here? The schema for rollback arguments [doesn't have a `slug` field](https://searchfox.org/mozilla-central/rev/ffdb6a4d934b3f5294f18cf0e1df618109ccb36b/toolkit/components/normandy/actions/schemas/index.js#112-124), it has `rolloutSlug`. Is the server returning errors on both?

File: content/components/recipes/form/arguments/MultiPreferenceBranches.js [hunk 2]
```diff
@@ -230,6 +231,7 @@ function Branch({ index }) {
     return parseInt(value, 10);
   };
 
+  const errorPrefix = `arguments.branches.${index}.`;
```
Comment (mythmon) [inline content/components/recipes/form/arguments/MultiPreferenceBranches.js:235]: There is also room to have errors specifically on `arguments.branches`, for example if no branches are added. I think we should have an error display for that as well.

**Review by mythmon — DISMISSED** [review]

**Comment by mythmon** [comment]
I also noticed that changes like switching the action type when creating a recipe don't clear out the errors, so you get some non-sensical errors in that case. I think that's fine though, since it is probably a bigger issue, and it is an uncommon case anyways.

**Comment by leplatrem** [comment]
Thanks @mythmon for pointout the action arguments schemas! That's helpful indeed :)

I can look to clean-up the form error on action type switching in a separate PR

**Comment by leplatrem** [comment]
I'm not super happy with the appearance... 

Screenshot from 2020-12-11 16-56-20

I'll move it below...

**Pull request merged**